### PR TITLE
added a highlight blue used by concierge and the header component and a pink for concierge

### PR DESCRIPTION
**Pull request by dipunm**

### CODE REVIEW

File: OTTheme/ottheme-colors/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ottheme-colors",
-  "version": "1.1.2",
+  "version": "1.1.3",
```
Comment (matteofigus) [inline OTTheme/ottheme-colors/package.json]: please can you revert the version change, we handle this during publish with lerna

**Comment by nickbalestra** [comment]
@matteofigus is good for me, I leave it to you to merge/publish

File: OTTheme/ottheme-colors/package.json (hunk 1 above)
Comment (dipunm) [reply OTTheme/ottheme-colors/package.json]: I reverted the change

**Comment by dipunm** [comment]
Guys, I am adding one more colour. I'll message here when it's done.

**Comment by dipunm** [comment]
added color-misty-rose.

**Pull request merged**